code/script.js

```diff
@@ -26,22 +32,117 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+
   chat.scrollTop = chat.scrollHeight
 }
 
+//This function makes the conversation happen
+const conversation = (message) => {
+  if (questionNumber === 1) {
+    setTimeout(() => greeting(message, 'bot'), 1000)
+
+  } else if (questionNumber === 2) {
+    showMessage(message, 'user') 
+    handleNameInput.value = ''
+    setTimeout(() => drinks(message, 'user'), 1000)
+
+  } else if (questionNumber === 3) {
+    showMessage(message, 'user')
+    setTimeout(() => drinkChoices(message), 1000)
+
+  } else (questionNumber === 4)
```

Comment:
We were curious at first: What's question nr. 4? We know you define it further down below, but maybe adding a comment in here could also help? Just for extra clarity :)